starter-bots/scala/src/main/scala/za/co/entelect/challenge/Main.scala

```diff
@@ -0,0 +1,49 @@
+package za.co.entelect.challenge
+
+import java.nio.file.{Files, Paths}
+
+import org.json4s.DefaultFormats
+
+import scala.io.StdIn.readInt
+import scala.util.Random
+import org.json4s._
+import org.json4s.native.JsonMethods._
+import za.co.entelect.challenge.commands.Command
+import za.co.entelect.challenge.entities.GameState
+import za.co.entelect.challenge.enums.Serializers
+
+object Main {
+  private final val ROUNDS_DIRECTORY = "rounds"
+  private final val STATE_FILE_NAME  = "state.json"
```

Comment:
`final` is not necessary here. The reason for this is that `final` prevents the fields from being overridden. These fields are both private and in a singleton object, so they cannot be overridden 🙂